src/components/InjectedComponents/PostDialog.tsx

```diff
@@ -16,12 +16,15 @@ import {
     DialogProps,
     Tooltip,
 } from '@material-ui/core'
+import Jimp from 'jimp'
+import type JimpT from 'jimp'
+var toBuffer = require('typedarray-to-buffer') // TODO: !
```

Comment:
Hmm todo what?

Reply:
```
Buffer.from(uint8array)
```

it work

Reply:
I have never used that Node.js style import in a React project before. But I have read that this gets converted into the right syntax with babel. Right?

Reply:
webpack bundler include `Buffer` polyfill.
see https://v4.webpack.js.org/configuration/node/

Reply:
but webpack 5 removed the support
see https://github.com/webpack/changelog-v5#changes-to-the-structure

Reply:
but, note `.slice(...)`  is reference parent buffer

```plain
> payload = Buffer.from('example-1', 'ascii')
<Buffer 65 78 61 6d 70 6c 65 2d 31>
> sliced = payload.slice(8)
<Buffer 31>
> sliced.write('2')
1
> payload.toString()
'example-2'
>
```